### ISSUE TITLE
[bechdelbot] Handle common user entry "mistakes"

### ISSUE THREAD

**Issue by shannonturner** [shannonturner/bechdel#34]
Bechdelbot won't give a response to a tweet unless it can find the movie the user asked for.  That's the preferred behavior, but often, people will include extra punctuation (for example) as part of the tweet, and Bechdelbot thinks this is part of the movie's title, doesn't find it, and doesn't reply.

Example:
(at) bechdelbot jaws?

will fail because there's no question mark in the title of the movie.  But this has been a common point of failure.

tl;dr: .strip() out question marks from the tweets
